vscode/features.json5

```
@@ -0,0 +1,102 @@
+{
+    "editors": ["vscode", "jetbrains"],
+    "features": [
+        {
+        "name": "NotebookBasedChatUI",
+        "description": "new chat UI that resembles a notebook instead of a chat session",
```

```suggestion
        "description": "Chat UI that resembles a Jupyter notebook instead of a chat session",
```

- Let's avoid "new" because that will become inaccurate over time.
- I'm mentioning Jupyter here so people don't get confused with a paper notebook.